### PR TITLE
Editorial update suggestions for Privacy Considerations section

### PR DESCRIPTION
**Pull request by grantnoble**
A lot of "verifiable" credentials, some grammar and punctuation

***
<a href="https://pr-preview.s3.amazonaws.com/grantnoble/vc-data-model/pull/527.html" title="Last updated on Apr 4, 2019, 5:31 AM UTC (c65b357)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/527/1ad7698...grantnoble:c65b357.html" title="Last updated on Apr 4, 2019, 5:31 AM UTC (c65b357)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3587,21 +3588,22 @@ <h3>Favor Abstract Claims</h3>
 To enable recipients of <a>verifiable credentials</a> to use them in a variety
 of circumstances without revealing more personally identifiable information
 than necessary for transactions, <a>issuers</a> should consider limiting the
-information published in a <a>credential</a> to a minimal set needed for the
-expected purposes. One way to avoid placing personally identifiable information
-in a <a>credential</a> is to use an abstract <a>property</a> that meets the
-needs of <a>verifiers</a> without providing specific information about a
-<a>subject</a>.
+information published in a <a>verifiable credential</a> to a minimal set needed
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 2]
```diff
@@ -3587,21 +3588,22 @@ <h3>Favor Abstract Claims</h3>
...
+for the expected purposes. One way to avoid placing personally identifiable
+information in a <a>verifiable credential</a> is to use an abstract
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 3]
```diff
@@ -3852,33 +3853,34 @@ <h3>Aggregation of Credentials</h3>
...
+For example, if two <a>bearer credentials</a>, one for an email address and then
 one stating the holder is over the age of 21, are provided across multiple
 sessions, the <a>verifier</a> of the information now has a unique identifier
-plus age-related information for the individual. It is now easy to create and
-build a profile for the <a>holder</a> such that more and more information is
-leaked over time. Aggregation of <a>credentials</a> can also be performed across
-multiple sites in collusion with each other, leading to privacy violations.
+as well as age-related information for that individual. It is now easy to create
+and build a profile for the <a>holder</a> such that more and more information is
+leaked over time. Aggregation of <a>verifiable credentials</a> can also be
```
Comment (msporny) [inline index.html]: nope, credentials.

File: index.html [hunk 4]
```diff
@@ -3896,44 +3898,44 @@ <h3>Usage Patterns</h3>
...
-The underlying information in a <a>credential</a> can be used to identify an
-individual across services. In this case, using information from other sources
-(including information provided directly by the <a>holder</a>),
-<a>verifiers</a> can use information inside the <a>credential</a> to correlate
-the individual with an existing profile. For example, if a <a>holder</a>
-presents <a>credentials</a> that include postal code, age, and gender, a
-<a>verifier</a> can potentially correlate the <a>subject</a> of that
-<a>credential</a> with an established profile. For more information,
-see [[DEMOGRAPHICS]].
+The underlying information in a <a>verifiable credential</a> can be used to
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 5]
```diff
@@ -3896,44 +3898,44 @@ <h3>Usage Patterns</h3>
...
+identify an individual across services. In this case, using information from
+other sources (including information provided directly by the <a>holder</a>),
+<a>verifiers</a> can use information inside the <a>verifiable credential</a> to
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 6]
```diff
@@ -3896,44 +3898,44 @@ <h3>Usage Patterns</h3>
...
+information, see [[DEMOGRAPHICS]].
         </li>
         <li>
-Passing the identifier of a <a>credential</a> to a centralized revocation
-server. In this case the centralized server can correlate the <a>credential</a>
-usage across interactions. For example, if a <a>verifiable credential</a> is
-used for proof of age in this manner, the centralized service could know
-everywhere that credential was presented: all liquor stores, bars, adult
-stores, lottery purchases, and so on.
+Passing the identifier of a <a>verifiable credential</a> to a centralized
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 7]
```diff
@@ -3945,15 +3947,16 @@ <h3>Usage Patterns</h3>
       <ul>
         <li>
 Using a globally-unique identifier as the <a>subject</a> for any given
-<a>credential</a> and never re-use that <a>credential</a>.
+<a>verifiable credential</a> and never re-use that <a>verifiable credential</a>.
```
Comment (msporny) [inline index.html]: credential, in both cases.

File: index.html [hunk 8]
```diff
@@ -3945,15 +3947,16 @@ <h3>Usage Patterns</h3>
...
         </li>
         <li>
-If the <a>credential</a> supports revocation, using a globally-distributed
-service for revocation.
+If the <a>verifiable credential</a> supports revocation, using a
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 9]
```diff
@@ -3945,15 +3947,16 @@ <h3>Usage Patterns</h3>
...
         <li>
-If the <a>credential</a> supports revocation, using a globally-distributed
-service for revocation.
+If the <a>verifiable credential</a> supports revocation, using a 
+globally-distributed service for revocation.
         </li>
         <li>
 Designing revocation APIs that do not depend on submitting the ID of the
-<a>credential</a>. For example, use a revocation list instead of a query.
+<a>verifiable credential</a>. For example, use a revocation list instead of a
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 10]
```diff
@@ -3983,8 +3986,9 @@ <h3>Usage Patterns</h3>
       </p>
 
       <p>
-Privacy risks of <a>credential</a> usage occur when unintended or unexpected
-correlation arises from the presentation of <a>verifiable credentials</a>.
+Privacy risks of <a>verifiable credential</a> usage occur when unintended or
```
Comment (msporny) [inline index.html]: should probably be credential in both cases.

File: index.html [hunk 11]
```diff
@@ -3983,8 +3986,9 @@ <h3>Usage Patterns</h3>
...
+unexpected correlation arises from the presentation of
+<a>verifiable credentials</a>.
```
Comment (msporny) [inline index.html]: should probably be credential. Whatever you change it to, make sure the sentence uses the same form. Prefer the broader point being made about "credentials".

File: index.html [hunk 12]
```diff
@@ -4035,22 +4039,24 @@ <h3>Frequency of Claim Issuance</h3>
...
       </p>
 
       <p>
 Organizations providing software to <a>holders</a> should warn them if they
-repeatedly use <a>credentials</a> with short lifespans, which could result in
-behavior correlation. <a>Issuers</a> should avoid issuing <a>credentials</a> in
-a way that enables them to correlate usage patterns.
+repeatedly use <a>verifiable credentials</a> with short lifespans, which could
+result in behavior correlation. <a>Issuers</a> should avoid issuing
+<a>verifiable credentials</a> in a way that enables them to correlate usage
```
Comment (msporny) [inline index.html]: nope, credentials.

File: index.html [hunk 13]
```diff
@@ -4035,22 +4039,24 @@ <h3>Frequency of Claim Issuance</h3>
...
+renewal mechanism, then the <a>issuer</a> could possibly correlate the behavior
+of the <a>holder</a> in a way that negatively impacts the <a>holder</a>.
       </p>
 
       <p>
 Organizations providing software to <a>holders</a> should warn them if they
-repeatedly use <a>credentials</a> with short lifespans, which could result in
-behavior correlation. <a>Issuers</a> should avoid issuing <a>credentials</a> in
-a way that enables them to correlate usage patterns.
+repeatedly use <a>verifiable credentials</a> with short lifespans, which could
```
Comment (msporny) [inline index.html]: nope, credentials.

**Comment by msporny** [comment]
Also, merge conflicts. :(

File: index.html (hunk 1 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 4 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 5 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 6 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 7 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 8 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 9 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 10 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 11 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 12 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 13 above)
Comment (grantnoble) [reply index.html]: Fixed.

**Comment by grantnoble** [comment]
Merge conflicts resolved.

File: index.html [hunk 14]
```diff
@@ -3508,15 +3508,15 @@ <h3>Identifier-Based Correlation</h3>
...
-where multiple <a>verifiers</a>, or an <a>issuer</a> and a <a>verifier</a>,
-can collude to correlate the <a>holder</a>. If <a>holders</a> want to reduce
-correlation, they should use <a>credential</a> schemes that allow hiding the
-identifier during presentation. Such schemes expect the <a>holder</a> to
+Similarly, disclosing the <a>verifiable credential</a> identifier
+(<code>credential.id</code>) leads to situations where multiple
+<a>verifiers</a>, or an <a>issuer</a> and a <a>verifier</a>, can collude to
+correlate the <a>holder</a>. If <a>holders</a> want to reduce correlation, they
+should use <a>verifiable credential</a> schemes that allow hiding the
+identifier during <a>presentation</a>. Such schemes expect the <a>holder</a> to
```
Comment (msporny) [inline index.html]: "presentation" -> "verifiable presentation"

File: index.html [hunk 15]
```diff
@@ -3508,15 +3508,15 @@ <h3>Identifier-Based Correlation</h3>
       </p>
 
       <p>
-Similarly, disclosing the <a>credential</a> identifier
-(<code>credential.id</code>) leads to situations
-where multiple <a>verifiers</a>, or an <a>issuer</a> and a <a>verifier</a>,
-can collude to correlate the <a>holder</a>. If <a>holders</a> want to reduce
-correlation, they should use <a>credential</a> schemes that allow hiding the
-identifier during presentation. Such schemes expect the <a>holder</a> to
+Similarly, disclosing the <a>verifiable credential</a> identifier
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 16]
```diff
@@ -3571,14 +3571,15 @@ <h3>Long-Lived Identifier-Based Correlation</h3>
...
+<a>verifiable credential</a>-specific JSON-LD contexts, and many other sorts of
+long-lived identifiers.
       </p>
 
       <p>
 Organizations providing software to <a>holders</a> should strive to identify
-fields in <a>credentials</a> containing information that could be used to
-correlate individuals and warn <a>holders</a> when this information is shared.
+fields in <a>verifiable credentials</a> containing information that could be
+used tocorrelate individuals and warn <a>holders</a> when this information is
```
Comment (msporny) [inline index.html]: typo "tocorrelate"

File: index.html [hunk 17]
```diff
@@ -3816,36 +3817,36 @@ <h3>Validity Checks</h3>
...
-<a>Issuers</a> are urged to not use mechanisms, such as <a>credential</a>
-revocation lists that are unique per <a>credential</a>, during the
-<a>verification</a> process that could lead to privacy violations.
-Organizations providing software to <a>holders</a> should warn when
-<a>credentials</a> include information that could lead to privacy violations
-during the verification process. <a>Verifiers</a> should consider rejecting
-<a>credentials</a> that produce privacy violations or that enable bad privacy
-practices.
+<a>Issuers</a> are urged to not use mechanisms, such as
+<a>verifiable credential</a> revocation lists that are unique per
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 18]
```diff
@@ -3816,36 +3817,36 @@ <h3>Validity Checks</h3>
...
-revocation lists that are unique per <a>credential</a>, during the
-<a>verification</a> process that could lead to privacy violations.
-Organizations providing software to <a>holders</a> should warn when
-<a>credentials</a> include information that could lead to privacy violations
-during the verification process. <a>Verifiers</a> should consider rejecting
-<a>credentials</a> that produce privacy violations or that enable bad privacy
-practices.
+<a>Issuers</a> are urged to not use mechanisms, such as
+<a>verifiable credential</a> revocation lists that are unique per 
+<a>verifiable credential</a>, during the <a>verification</a> process that could
```
Comment (msporny) [inline index.html]: nope, credential.

File: index.html [hunk 19]
```diff
@@ -3816,36 +3817,36 @@ <h3>Validity Checks</h3>
...
+lead to privacy violations. Organizations providing software to <a>holders</a>
+should warn when <a>verifiable credentials</a> include information that could
```
Comment (msporny) [inline index.html]: nope, credentials.

File: index.html [hunk 20]
```diff
@@ -3816,36 +3817,36 @@ <h3>Validity Checks</h3>
...
+lead to privacy violations during the verification process. <a>Verifiers</a>
+should consider rejecting <a>verifiable credentials</a> that produce privacy
```
Comment (msporny) [inline index.html]: nope, credentials.

File: index.html [hunk 21]
```diff
@@ -3922,44 +3923,43 @@ <h3>Usage Patterns</h3>
...
+<a>verifiers</a>, and either those <a>verifiers</a> collude, or a third party
+has access to transaction records from both <a>verifiers</a>. An observant
+party could infer that the individual presenting the
+<a>verifiable credential</a> is the same person at both services. That is, the
+accounts are controlled by the same person.
         </li>
         <li>
-A <a>subject</a> identifier of a <a>credential</a> refers to the same
+A <a>subject</a> identifier of a <a>verifiable credential</a> refers to the same
 <a>subject</a> across multiple <a>presentations</a> or <a>verifiers</a>. Even
```
Comment (msporny) [inline index.html]: If you're going to change this section to use "verifiable credentials", then all "presentations" need to change to be "verifiable presentation" as well.

File: index.html (hunk 14 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 15 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 16 above)
Comment (grantnoble) [reply index.html]: Good catch!

File: index.html (hunk 17 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 19 above)
Comment (grantnoble) [reply index.html]: Fixed.

File: index.html (hunk 20 above)
Comment (grantnoble) [reply index.html]: fixed.

File: index.html (hunk 21 above)
Comment (grantnoble) [reply index.html]: changed back to "credential"

File: index.html (hunk 18 above)
Comment (grantnoble) [reply index.html]: Fixed.

**Pull request merged**